CONTRIBUTING.md

```diff
@@ -33,16 +33,16 @@ using the `jetbrains-ide` & `team/integrations` labels.
 | Reformat Java and Kotlin sources                              | `./gradlew spotlessApply`                                                |
 | Debug agent JSON-RPC communication                            | `tail -f build/sourcegraph/cody-agent-trace.json`                        |
 
-## Using Alpha channel releases
+## Using Nightly channel releases
 
-We occasionally publish plugins to the "Alpha" channel instead of the default
-"Stable" channel. The alpha channel is primarily intended to publish
+We occasionally publish plugins to the "Nightly" channel instead of the default
+"Stable" channel. The nightly channel is primarily intended to publish
 pre-releases for internal (within Sourcegraph) testing.
 
 - Open Settings
 - Open "Plugins"
 - Click on cogwheel in the top bar, select "Manage plugin repositories"
-- Add the URL https://plugins.jetbrains.com/plugins/list?channel=alpha&pluginId=9682
+- Add the URL https://plugins.jetbrains.com/plugins/list?channel=nightly&pluginId=9682
```

Comment:
nit: We can use:
https://plugins.jetbrains.com/plugins/nightly/9682
here too!

Reply:
I'll leave this as is because I want to merge the PR. This documentation can be replaced with "Enable nightly channel in settings" once it's live